[2021-03-12 11:11:21,792] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: conversion_flow.taskPrepare 2021-03-12T11:11:20+00:00 [queued]>
[2021-03-12 11:11:21,801] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: conversion_flow.taskPrepare 2021-03-12T11:11:20+00:00 [queued]>
[2021-03-12 11:11:21,801] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-12 11:11:21,801] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-12 11:11:21,801] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-12 11:11:21,806] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): taskPrepare> on 2021-03-12T11:11:20+00:00
[2021-03-12 11:11:21,808] {standard_task_runner.py:52} INFO - Started process 1298 to run task
[2021-03-12 11:11:21,810] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'conversion_flow', 'taskPrepare', '2021-03-12T11:11:20+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', '/srv/app/src/ckanext-mysql2mongodb/ckanext/mysql2mongodb/data_conv/dag/conversion_flow.py', '--cfg-path', '/tmp/tmp3cxu5uw7', '--error-file', '/tmp/tmpr8spblfk']
[2021-03-12 11:11:21,811] {standard_task_runner.py:77} INFO - Job 8: Subtask taskPrepare
[2021-03-12 11:11:21,834] {logging_mixin.py:104} INFO - Running <TaskInstance: conversion_flow.taskPrepare 2021-03-12T11:11:20+00:00 [running]> on host 6299f34f884f
[2021-03-12 11:11:21,856] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=conversion_flow
AIRFLOW_CTX_TASK_ID=taskPrepare
AIRFLOW_CTX_EXECUTION_DATE=2021-03-12T11:11:20+00:00
AIRFLOW_CTX_DAG_RUN_ID=conversion_flow1615547480.130667
[2021-03-12 11:11:22,002] {python.py:118} INFO - Done. Returned value was: True
[2021-03-12 11:11:22,015] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=conversion_flow, task_id=taskPrepare, execution_date=20210312T111120, start_date=20210312T111121, end_date=20210312T111122
[2021-03-12 11:11:22,032] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-12 11:11:22,062] {local_task_job.py:146} INFO - Task exited with return code 0
